workinggroups/README.md

```diff
@@ -0,0 +1,11 @@
+# CNCF Working Groups
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
```

Comment:
It is very important that every WG be created with a charter that explicitly states the mission of the WG, what its concrete deliverables are expected to be, and what it's exit criteria are. WGs should exist forever. They should address specific issues or questions.

Reply:
And if they publish information under the CNCF, such as landscapes or whitepapers, I think it makes sense to at least send them to the TOC for review, possibly with an explicit time bound for feedback ("RFC" period).

Reply:
Presumably you mean "WG's should NOT exist forever"?

Reply:
> And if they publish information under the CNCF, such as landscapes or whitepapers, I think it makes sense to at least send them to the TOC for review, possibly with an explicit time bound for feedback ("RFC" period).

Should it be implicit or explicit? For example, is lazy-concensus ok?